### ISSUE TITLE
Dropdown to always show empty as the first value, instead of "Please select:"

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#6389]
For the current behavior, and argument for making this change watch the following video.

https://github.com/orbeon/orbeon-forms/assets/106961/61f8d4c3-5ec6-4e4d-b528-4268b811770b

[+1 from customer](https://3.basecamp.com/3600924/buckets/1956846/messages/7530218291)

**Comment by avernet** [comment]
That's a tough one. I'm not finding a way to have the dropdown show "Please select:" when closed, and when opened have the first choice be empty instead of "Please select...".

I have a prototype where "Please select:" is shown above the dropdown when the value is empty, but getting this to display properly across browsers and operating systems seems finicky.

**Comment by avernet** [comment]
Idea: Rely on `<option hidden selected>Please select:</option>` being shown in the dropdown when closed, but not when open. When no value is selected, the first item is `<option hidden selected>Please select:</option>`. However, when a value is selected, we change it to `<option/>`.

To implement this, we need to support `xxf:hidden="{...}"` on `<xf:item>`. This support should be added in the content handler, Ajax response, and XFormsUI.

**Comment by avernet** [comment]
> Idea: Rely on `<option hidden selected>Please select:</option>` being shown in the dropdown when closed, but not when open.

Bummer: that works with Chrome, but not with Safari. Back to the drawing board.

**Comment by avernet** [comment]
The two options that are both consistent and certain possible to implement without "hacks" with native dropdowns are to:

- The first entry to always be "Please select".
- The first entry to always be empty.

**Comment by avernet** [comment]
We decided to always display "Please select" instead of leaving the field empty. This ensures consistency and improves accessibility:

- Pragmatically, options without labels are often flagged by accessibility checkers.
- Practically, empty fields are not read clearly by VoiceOver or other assistive technologies.

[+1 from customer](https://3.basecamp.com/3600924/buckets/2192969/messages/9711482358)
